generators/entity/index.js

```diff
@@ -272,6 +288,14 @@ class EntityGenerator extends BaseBlueprintGenerator {
           this.log(`\nThe entity ${entityName} is being created.\n`);
         }
       },
+
+      reloadDerivedConfig() {
+        // May have been modified after options
+        this.loadDerivedAppConfig(_.defaults({}, this.jhipsterConfig), this.context);
+        this.loadDerivedClientConfig(_.defaults({}, this.microserviceConfig, this.jhipsterConfig), this.context);
+        // Try to load server config from microservice side, falling back to the app config.
+        this.loadDerivedServerConfig(_.defaults({}, this.microserviceConfig, this.jhipsterConfig), this.context);
+      },
```

Comment:
@Tcharl the entity generator crashes without this?
- The api usage looks wrong `loadDerivedAppConfig(dest, config)`, dest/config is inverted.
- server config is from `this.microserviceConfig` while client config should be loaded from the gateway `this.jhipsterConfig` (while microfrontend is not implemented).

We don't have Microservices tests in our main repository, this is easy to break.
This will get easier when microfrontend lands.
```suggestion
```

Reply:
I use the same syntax as l#227. Anyway, after investigation, there's no option that can impact derived property: I find it a little bit risky, but this code can disappear